src/Places/index.ts

```diff
@@ -165,6 +165,12 @@ export function getPlaceDestination(place: Place, parentPlacesMap: Map<string, P
 		}
 	}
 
+	const countryParentId: string|undefined = reversedPlaceParentIds.find((parentId) => parentId.includes('country:'));
```

Comment:
Noooo this is exactly what we don'r want to do. We have a to work with ids like it is string... There is apossibility that with some new API version all wil get some random hash id. You have to use value from level property for this.